root - ERROR - Exception occured - 2019-11-29 17:07:31,055 - 12 - scientific_calc - sine_h - build\bdist.win-amd64\egg\src\driver\scientific_calc.py
None
